docs/i18n/en.yaml

```diff
@@ -0,0 +1,17 @@
+- id: get-started
```

Comment:
This file isn't changed from the theme file so if you remove it here it should get picked up from there.